test/integration/controller/core/workload_controller_test.go

```diff
@@ -63,6 +64,7 @@ var _ = ginkgo.Describe("Workload controller", func() {
 	ginkgo.AfterEach(func() {
```

Comment:
Not sure why we were doing this. It doesn't seem necessary to set everything to nil.

Reply:
I added flavor to the top of the ginkgo spec since I tested the `ClusterQueueStatus.Conditions` in `workload_controller_test.go`.

Although we can not remove these flavors since we ignore the `ClusterQueueStatus.Conditions` as you mentioned in the following:

https://github.com/kubernetes-sigs/kueue/pull/415#discussion_r1046096792

```
[test/integration/controller/core/workload_controller_test.go](https://github.com/kubernetes-sigs/kueue/pull/415/files/0a7de4beee27755ac5815e234a03dd02acbeb5c1#diff-3c7d6a020caf54db44c9cf146bce6c049e9124d717c6ec38827c108591ac3c23)
Outdated
						Message: "Can admit new workloads",
					},
				},
			}, ignoreCQConditionTimestamps))
Member
@[alculquicondor](https://github.com/alculquicondor) alculquicondor [last week](https://github.com/kubernetes-sigs/kueue/pull/415#discussion_r1046096792)
just ignore the conditions as well. This is not where we should be testing that.

Member
Author
@[tenzen-y](https://github.com/tenzen-y) tenzen-y [4 days ago](https://github.com/kubernetes-sigs/kueue/pull/415#discussion_r1047997766)
Makes sense.

Member
Author
@[tenzen-y](https://github.com/tenzen-y) tenzen-y [1 hour ago](https://github.com/kubernetes-sigs/kueue/pull/415#discussion_r1051629530)
Done.
```

Reply:
Done.